spec/fixtures/solr_cores/hyacinth-solr-6-3/conf/solrconfig.xml

```diff
@@ -748,13 +748,33 @@
        of SearchComponents (see below) and supports distributed
        queries across multiple shards
     -->
+  <!-- request handler to return searches -->
+  <requestHandler name="/search" class="solr.SearchHandler">
+    <lst name="defaults">
+      <str name="echoParams">explicit</str>
+      <str name="defType">edismax</str>
+      <str name="q">*:*</str>
+      <str name="q.alt">*:*</str>
+      <str name="rows">10</str>
+      <str name="qf">id^500, digital_object_type_ssi^40 title_ssi^30</str>
```

Comment:
Why is digital_object_type_ssi in here?  I don't remember if this is a thing carried over from Hyacinth 2, and if so, why we're including it there.

Reply:
those are just the existing fields in a proposed score

Reply:
```suggestion
      <str name="qf">id^500, title_ssi^30</str>
```